ci-operator/config/openstack-k8s-operators/designate-operator/openstack-k8s-operators-designate-operator-main.yaml

```diff
@@ -0,0 +1,31 @@
+build_root:
+  from_repository: true
+images:
+- dockerfile_literal: |
+    FROM src
+    RUN pip3 install pre-commit
+  from: src
+  to: openstack-operator-ci-pre-commit
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
```

Comment:
make gotest is not covered by pre-commit, can we can as  job here?
```
- as: unit
  commands: make gotest
  container:
    from: src
```

Reply:
Hey Doug,

Designate operator don't have any gotests currently, I requested bhagyashri to drop this job and we can readd once we have the gotests.